src/dos/program_mount.cpp

```diff
@@ -285,15 +284,14 @@ void MOUNT::Run(void) {
 			}
 
 			int error = 0;
-			newdrive  = std::make_unique<cdromDrive>(
-			        drive,
-			        temp_line.c_str(),
-			        sizes[0],
-			        int8_tize,
-			        sizes[2],
-			        0,
-			        mediaid,
-			        error);
+			newdrive  = std::make_shared<cdromDrive>(drive,
+			                                         temp_line.c_str(),
+			                                         sizes[0],
+			                                         int8_tize,
+			                                         sizes[2],
+			                                         0,
+			                                         mediaid,
+			                                         error);
```

Comment:
Consider extracting the args to well-named constants; I have no idea what all this crap means 😆

Reply:
I don't either, honestly.  Mostly all I did in this file was change from `unique_ptr` to `shared_ptr`.  I'm gonna save that for another day because I would probably have to refactor that entire large function to figure out what the heck is actually going on.

Reply:
Sure, it's no worse than before. That's something 😆